testlist/arm-01.dat

```diff
@@ -6,6 +6,6 @@
 
 /arm/cxd56xx,CONFIG_ARMV7M_TOOLCHAIN_GNU_EABIL
 
-/arm/dm320,CONFIG_ARMV7M_TOOLCHAIN_GNU_EABIL
-
 /arm/efm32,CONFIG_ARMV7M_TOOLCHAIN_GNU_EABIL
+
+/arm/stm32f0l0g0,CONFIG_ARMV7M_TOOLCHAIN_GNU_EABIL
```

Comment:
Could we order the list? so we can find the untested board quickly.

Reply:
If in the alphabetic order completely, we should use more level depth boards path instead.

Reply:
Now refine in  alphabetic order done.